src/lammps.cpp

```diff
@@ -858,10 +859,33 @@ void LAMMPS::post_create()
 {
   if (skiprunflag) input->one("timer timeout 0 every 1");
 
+  // Don't unnecessarily reissue a package command via suffix
```

Comment:
The changes here break the `-pk pair/only on` flag. Unlike what is said in the comment, suffix *is* going to be NULL in that case (suffixp isn't) and thus this causes a segfault. I am looking into a workaround, but have not yet found one.

Reply:
Workaround should be available now. This also fixes a related bug that has crept in unnoticed.